src/app/technical-record/body/body.component.spec.ts

```diff
@@ -1,59 +1,41 @@
 import { async, ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
-import { StoreModule } from '@ngrx/store';
 import { TechRecordHelpersService } from '@app/technical-record/tech-record-helpers.service';
-import { appReducers } from '@app/store/reducers/app.reducers';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { MatDialogModule } from '@angular/material/dialog';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MaterialModule } from '@app/material.module';
 import { SharedModule } from '@app/shared/shared.module';
-import { RouterTestingModule } from '@angular/router/testing';
-import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
-import { NgrxFormsModule } from 'ngrx-forms';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { VehicleSummaryComponent } from '@app/technical-record/vehicle-summary/vehicle-summary.component';
-import { TechnicalRecordComponent } from '@app/technical-record/technical-record.component';
+import { BodyComponent } from '@app/technical-record/body/body.component';
 
-describe('VehicleSummaryComponent', () => {
-  let component: VehicleSummaryComponent;
-  let fixture: ComponentFixture<VehicleSummaryComponent>;
+describe('BodyComponent', () => {
+  let component: BodyComponent;
+  let fixture: ComponentFixture<BodyComponent>;
   let injector: TestBed;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StoreModule.forRoot(appReducers),
-        HttpClientTestingModule,
-        MatDialogModule,
-        FormsModule,
-        ReactiveFormsModule,
-        BrowserAnimationsModule,
-        MaterialModule,
-        SharedModule,
-        RouterTestingModule,
-        FontAwesomeModule,
-        ReactiveFormsModule,
-        NgrxFormsModule
-      ],
-      declarations: [VehicleSummaryComponent, TechnicalRecordComponent],
+      imports: [HttpClientTestingModule, SharedModule],
+      declarations: [BodyComponent],
       providers: [TechRecordHelpersService],
       schemas: [CUSTOM_ELEMENTS_SCHEMA]
     }).compileComponents();
 
-    fixture = TestBed.createComponent(VehicleSummaryComponent);
+    fixture = TestBed.createComponent(BodyComponent);
     injector = getTestBed();
     component = fixture.componentInstance;
     component.activeRecord = {
```

Comment:
Can we use a mock for that please?
I think @m-ujaffer has created a few utils functions for it